src/server/components/dashboard/listsItems/item/update/actionHandlers/index.ts

```diff
@@ -0,0 +1,20 @@
+import { Action } from "../types";
+import { update } from "./update";
+import { pin } from "./pin";
+import { remove } from "./remove";
+import { publish } from "./publish";
+import { requestChanges } from "./requestChanges";
+import { RequestHandler } from "express";
+
+export const actionHandlers: Record<Action, RequestHandler> = {
```

Comment:
the guide is "what feels right" :p 

When you do an import, and don't specify the file, node will import `index.t|js`. So you can use index.ts to control what you do/don't want to expose. In this case, I want to just return a (typed) object. 

Inside the actionHandlers directory.. I have all of the route handlers that handle an action. they are fully qualified route handlers - they accept (req, res, next) and return a response (or calls next). Each of the action route handlers won't "help" with anything. 

If you take the case of requestChanges.ts, there is a helper function in there, but it isn't being exposed via helpers.ts or export. Essentially it's a private function - because this function doesn't help any of the other route handlers. When it does, it can be refactored into helpers.ts

Most of our "controllers" are just route handlers. I would expect a controller (class or interface) to serve more than one route handler. for example 
`usersEditController` and `usersEditPostController`. These are get and post route handlers respectively. It's the "usersEditController" - which serves the get and post route handlers, `usersEditController.get(req, res, next)`